espn_api/football/fantasy.py

```diff
@@ -0,0 +1,172 @@
+from espn_api.football import League
+from operator import attrgetter
+
+# Add award to dict of teams
+def award(team_name, award_string):
+	if awards.get(team_name) != None:
+		awards[team_name].append(award_string)
+	else: 
+		awards[team_name] = [award_string]
```

Comment:
You could use defaultdict to instantiate your dictionary, which would let you only need to do the append part here https://docs.python.org/2/library/collections.html#collections.defaultdict